### PR TITLE
chore: [sc-25287] Change .gitignore files to be consistent

### PR DESCRIPTION
**Pull request by jazz-b**
Story details: https://app.shortcut.com/wmfs/story/25287

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -84,9 +85,6 @@ jspm_packages
 # Webstorm
 .idea
 
-# VS Code
-.vscode
```
Comment (reecebrend) [inline .gitignore:88]: Any reason you've removed the .vscode directory from the ignore?

Comment (jazz-b) [reply .gitignore:88]: I just copied a .gitignore file from another plugin project.

Comment (jazz-b) [reply .gitignore:88]: The old .gitignore file wasn't ignoring /package-lock.json (which is what I was looking for).

**Review by reecebrend — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (jazz-b) [reply .gitignore]: I've restored the .vscode exclusion.

**Pull request merged**